projects/max14906/src/examples/basic/basic_example.c

```diff
@@ -71,47 +71,47 @@ int basic_example_main()
 	enum max14906_climit climit = MAX14906_CL_300;
 	enum max14906_climit climit2 = MAX14906_CL_600;
 
-	/** MAX14906 Initialization */
-	ret = max14906_init(&max14906_desc, &max14906_ip);
-	if (ret)
-		goto exit;
+	struct no_os_callback_desc gpio_cb = {
+		.callback = gpio_callback_fn,
+		/** Parameter to be passed when the callback is called. */
+		.ctx = max14906_desc,
+		.event = NO_OS_EVT_GPIO,
+		.peripheral = NO_OS_GPIO_IRQ,
+		.handle = NULL
+	};
 
 	/** Global Interrupt Controller */
```

Comment:
I think the comment should also be updated accordingly

Reply:
Updating it.